src/view/observer/focusobserver.js

```diff
@@ -8,6 +8,10 @@
  */
 
 import DomEventObserver from './domeventobserver';
+import global from '@ckeditor/ckeditor5-utils/src/dom/global';
+
+const setTimeout = global.window.setTimeout;
+const clearTimeout = global.window.clearTimeout;
```

Comment:
BTW, I haven't checked the code precisely, but IMO there's no point in using `global` if we're not stubbing something in the tests later. This module was initially created in order to allow stubbing globals.

Unless, of course, we decide that we should never use `window` and `document` directly. But it means that we'd need to change this in maaany places.